### PR TITLE
Add test for 400ZR Q-value performance monitoring

### PR DESCRIPTION
**Pull request by ahsaanyousaf**

### CODE REVIEW

File: feature/platform/transceiver/zr_db_q_value_test/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# TRANSCEIVER-6: Telemetry: 400ZR Optics Q-value streaming.
+
+## Summary
+
+Validate 400ZR optics module reports Q-value performance data as defined in
+module CMIS VDM(Versatile Diagnostics Monitor).
```
Comment (ejbrever) [inline feature/platform/transceiver/zr_db_q_value_test/README.md:6]: Is the term VDM vendor agnostic?

Comment (ejbrever) [reply feature/platform/transceiver/zr_db_q_value_test/README.md:6]: Thanks for pointing out, for some reason I imagined it being a vendor specific acronym :)

**Review by ejbrever — APPROVED** [review]
LGTM from me with Darren's comments addressed.

File: feature/platform/transceiver/zr_db_q_value_test/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    *   /terminal-device/logical-channels/channel/otn/state/q-value/instant
+    *   /terminal-device/logical-channels/channel/otn/state/q-value/avg
+    *   /terminal-device/logical-channels/channel/otn/state/q-value/min
+    *   /terminal-device/logical-channels/channel/otn/state/q-value/max
+
+*   For reported data check for validity min <= avg/instant <= max
+
+*   When the modules or the devices are still in a boot stage, they must not
+    stream any invalid string values like "nil" or "-inf" until valid values
+    are available for streaming.
```
[the remark this answers is not in the document]
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_db_q_value_test/README.md:33]: This is not an absolute power value which is typically specified in dBm. The units are actually dB and 0 in this case is the correct value that should be streamed when interface is in down state.

File: feature/platform/transceiver/zr_db_q_value_test/README.md (hunk 1 above)
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_db_q_value_test/README.md:6]: You are right this is defined in the CMIS :)

File: feature/platform/transceiver/zr_db_q_value_test/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+*   Q-value must always be of type decimal64. When link interfaces are in down
+    state 0.0 must be reported as a valid default value.
+
+
+**Note:** For min, max, and avg values, 10 second sampling is preferred. If 
+          10 seconds is not supported, the sampling interval used must be
+          communicated.
```
Comment (dplore) [inline feature/platform/transceiver/zr_db_q_value_test/README.md:41]:
```suggestion
          specified by adding a deviation to the test.
```

**Review by dplore — DISMISSED** [review]
LGTM.  Just One minor suggestion/nit if you care to add it.  Deviations refer to a method we use in the test code to programmatically declare some alteration of the test required by a DUT.  These are typically tracked as bugs, but might be long lived issues in cases where it is infeasible to update.

**Comment by ahsaanyousaf** [comment]
> LGTM. Just One minor suggestion/nit if you care to add it. Deviations refer to a method we use in the test code to programmatically declare some alteration of the test required by a DUT. These are typically tracked as bugs, but might be long lived issues in cases where it is infeasible to update.

Done

**Review by dplore — APPROVED** [review]

**Pull request merged**